playground/src/main/java/mx/yellowme/youst/playground/domain/ChartType.kt

```diff
@@ -0,0 +1,14 @@
+package mx.yellowme.youst.playground.domain
+
+/**
+ * Created by adrianleyvasanchez on 29,September,2019
+ */
+enum class ChartType(name: String) {
+    BAR("Bar"),
+    LINE("Line"),
+    BUBBLE( "Bubble"),
+}
+
+fun getTypeByName(name: String): ChartType {
```

Comment:
I think there's no need for this method, unless you want to add a default value in case the method `valueOf()` throws an exception.